refinery/ui/source/js/data-set-import/services/import-confirmation-service.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
This should really be a component and not a service. Services should not be interacting with the view.  https://stackoverflow.com/questions/27398741/angularjs-controller-vs-service

Reply:
I think I'm getting a bit confused here. The link is comparing controllers vs. services. Is a component a broader term for a controller?

Reply:
No, a component is a simplified directive. For any dom related things (buttons, modals, etc), they should be components. 
 
"Components are the basic UI building block of an Angular app"

"Services are typically used as the business logic of your application. Services are similar to singletons in the sense that they are created once, and the instance is maintained throughout the entire life cycle of your application. It is a good place to put your logical functions which many views or components will require, and also hold global cache which needs to be accessed throughout multiple areas in your application."

Reply:
https://docs.angularjs.org/guide/concepts

Reply:
Okay cool thank you for the clarification. Am I correct in saying that components are to be bundled with directives underneath the `xxxx/directives` directories? I haven't found anything in our codebase suggesting otherwise